.tx/config

```diff
@@ -3,7 +3,7 @@ host = https://www.transifex.com
 
 [qbittorrent.qbittorrent_master]
 file_filter = src/lang/qbittorrent_<lang>.ts
-lang_map = pt: pt_PT
+lang_map = pt: pt_PT, zh: zh_CN
```

Comment:
I think you missed line 22.

Reply:
Damn it. You're correct. Too much haste. Fixed now.